paddle/fluid/framework/details/build_strategy.cc

```diff
@@ -33,6 +38,15 @@ class ParallelExecutorPassBuilder : public ir::PassBuilder {
  public:
   explicit ParallelExecutorPassBuilder(const BuildStrategy &strategy)
       : ir::PassBuilder(), strategy_(strategy) {
+    // NOTE(dzh): memory optimize should be a runtime pass.
```

Comment:
Can at least be put after other passes and before multi_device_pass?

Reply:
done.